src/player/Player.php

```diff
@@ -271,6 +275,9 @@ public function __construct(Server $server, NetworkSession $session, PlayerInfo
 		$this->displayName = $this->username;
 		$this->locale = $this->playerInfo->getLocale();
 
+		if(($language = $server->getLanguageManager()->get($playerInfo->getLocale())) !== null)
```

Comment:
The server crashes if this is ever null. The language property must have a default to be used.

Reply:
This method can return NULL if the fallback language is missing, but this happens when the server starts, which causes an error and the server stops. Accordingly, if there is a spare language, the server will start successfully, and this method will not be able to return NULL